packages/legend-query-builder/src/stores/fetch-structure/tds/post-filter/QueryBuilderPostFilterStateBuilder.ts

```diff
@@ -178,6 +179,24 @@ const processPostFilterTree = (
   postFilterState: QueryBuilderPostFilterState,
   parentPostFilterNodeId: string | undefined,
 ): void => {
+  // NOTE: This checks if the expression is a simple function expression of Minus
```

Comment:
Felt like this can be handled at another layer right when we process simple function expression, since here what you're doing is just to take care of them at filter/post-filter level. I'll see what I can do here